examples/puppetmaster/vagrant/2_install_puppet_agent.sh

```diff
@@ -7,6 +7,14 @@ set -eou pipefail
 
 source utils.sh
 
+snapshot_name="$(agent_snapshot_name $PUPPET_AGENT_VERSION)"
+
+echo "Restoring snapshot '$snapshot_name' with agent v$PUPPET_AGENT_VERSION (if available)..."
+if ! vagrant snapshot restore "$snapshot_name"; then
```

Comment:
Oops, the `!` needs to be deleted. Should be:
```
if vagrant snapshot restore "$snapshot_name"; then
```

Reply:
fixed